docs/toc.md

```diff
@@ -61,7 +62,7 @@ There are also publications and lecture notes:
 * Videos at [Verification Corner](https://www.youtube.com/channel/UCP2eLEql4tROYmIYm5mA27A)
 
 And some books:
-* K. Rustan M. Leino, 2023, [_Program Proofs_](https://mitpress.mit.edu/9780262546232/program-proofs/), to be available in 2023.
+* K. Rustan M. Leino, 2023, [_Program Proofs_](https://mitpress.mit.edu/9780262546232/program-proofs/), available March, 2023.
```

Comment:
On the subsequent line, I suggest either removing the bullet about the draft version from Lulu or stating that it is available only until the MIT Press version comes out.

Reply:
Modified but left in, just until the book is for sure available.